lib/scenic/adapters/postgres/refresh_dependencies.rb

```diff
@@ -0,0 +1,67 @@
+module Scenic
+  module Adapters
+    class Postgres
+      class RefreshDependencies
+        def self.call(name, adapter, connection)
+          new(name, adapter, connection).call
+        end
+
+        def initialize(name, adapter, connection)
+          @name = name
+          @adapter = adapter
+          @connection = connection
+        end
+
+        def call
+          dependencies.each do |dependency|
+            adapter.refresh_materialized_view(dependency)
+          end
+        end
+
+        private
+
+        attr_reader :name, :adapter, :connection
+
+        DEPENDENCY_SQL = <<~SQL.freeze
```

Comment:
Same thing in Travis: https://travis-ci.org/thoughtbot/scenic/jobs/175636690#L267